src/components/interfaces/HMI_API.xml

```diff
@@ -503,6 +503,45 @@
     <element name="turnText"/>
 </enum>
 
+<enum name="TextFieldType">
+  <element name="mediaTitle">
+    <description>The data in this field contains the title of the currently playing audio track.</description>
+  </element>
+  <element name="mediaArtist">
+    <description>The data in this field contains the artist or creator of the currently playing audio track.</description>
+  </element>
+  <element name="mediaAlbum">
+    <description>The data in this field contains the album title of the currently playing audio track.</description>
+  </element>
+  <element name="mediaYear">
+    <description>The data in this field contains the creation year of the currently playing audio track.</description>
+  </element>
+  <element name="mediaGenre">
+    <description>The data in this field contains the genre of the currently playing audio track.</description>
+  </element>
+  <element name="mediaStation">
+    <description>The data in this field contains the name of the current source for the media.</description>
+  </element>
+  <element name="rating">
+    <description>The data in this field is a rating.</description>
+  </element>
+  <element name="currentTemperature">
+    <description>The data in this field is the current temperature.</description>
+  </element>
+  <element name="maximumTemperature">
+    <description>The data in this field is the maximum temperature for the day.</description>
+  </element>
+  <element name="minimumTemperature">
+    <description>The data in this field is the minimum temperature for the day.</description>
+  </element>
+  <element name="weatherTerm">
+    <description>The data in this field describes the current weather (ex. cloudy, clear, etc.).</description>
+  </element>
+  <element name="humidity">
+    <description>The data in this field describes the current humidity value.</description>
+  </element>
```

Comment:
@mfoedisch From a technical perspective: Create a fork of the sdl_evolution, create a branch named `bugfix/remove_none_element`, edit [the metadata types proposal](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0073-Adding-Metadata-Types.md) with the changes you wish to propose and commit your changes, then create a PR for this change.

All of this you can see from [the example I gave](https://github.com/smartdevicelink/sdl_evolution/pull/253) (except that the branch was created in the repository itself rather than a fork).

My apologies, but I really do not see what is unclear about the process.

This should not take much time at all. I understand that doing this may seem unnecessary, but we cannot go making changes to the implementations of these proposals without any sort of review. I believe that we have agreed that this wouldn't require a full review process, and could be approved in the same meeting that the change is brought up. In any case, I don't think anyone will dissent to this particular change.